Pod/Classes/NSURLSession/SessionMock.swift

```diff
@@ -36,4 +36,4 @@ enum SessionMockError: Error {
 /**
 A default delay to use when mocking requests
 */
-var DefaultDelay: Double = 0.25
+public var DefaultDelay: Double = 0.25
```

Comment:
I'm not sure if you intended to have this global variable as public, but it seems that Swift doesn't allow to use private vars on public functions as default values on parameters anymore.